config/routes.rb

```diff
@@ -1,5 +1,6 @@
 Rails.application.routes.draw do
 
+  get 'words/index'
```

Comment:
Since you already have resources for words. You don't need this route anymore, please remove